### PR TITLE
Logging

### LINKED ISSUE #102: Logging
In particular, how to get logging of SQL statements on non-dnx projects

### PR DESCRIPTION
**Pull request by rowanmiller**
Resolves #102

### CODE REVIEW

**Comment by rowanmiller** [comment]
Staged at http://ef.readthedocs.org/en/staging/miscellaneous/logging.html

**Comment by rowanmiller** [comment]
@ajcvickers could you take a quick look at this

File: docs/miscellaneous/logging/sample/EFLogging.ConsoleApp/Program.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+namespace EFLogging.ConsoleApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (var db = new BloggingContext())
+            {
+                var loggerFactory = ((IInfrastructure<IServiceProvider>)db).GetService<ILoggerFactory>();
```
Comment (ajcvickers) [inline docs/miscellaneous/logging/sample/EFLogging.ConsoleApp/Program.cs]: Do you have to cast to IInfrastructure here? Normally this is not needed?

**Comment by ajcvickers** [comment]
Looks good to me, although I am no expert on how the logging stuff is supposed to be used. :shipit:

**Pull request merged**